sdk/tests/common/compare_readers.rs

```diff
@@ -0,0 +1,185 @@
+// Copyright 2024 Adobe. All rights reserved.
```

Comment:
Is this a utility? I wonder if this should go in `sdk/tests/utils` to be more clear.

Reply:
Integration tests don't have access to unit test code. So I either need a separate copy or we would need to make some test code public.